sdcflows/transform.py

```diff
@@ -94,9 +103,9 @@ def fit(self, spatialimage):
         )
 
         # Cache
-        self.shifts = nb.Nifti1Image(vsm, spatialimage.affine, None)
-        self.shifts.header.set_intent("estimate", name="Voxel shift")
-        self.shifts.header.set_xyzt_units("mm")
+        self.mapped = nb.Nifti1Image(vsm, spatialimage.affine, None)
+        self.mapped.header.set_intent("estimate", name="Voxel shift")
+        self.mapped.header.set_xyzt_units("mm")
```

Comment:
Not an issue now, but may need to come back to this if we plan to change units to microns for small voxel sizes. For example, would it be better to set the units based on `spatialimage.header`?

Reply:
Yes, it would be better to pick it from `spatialimage.header`. I believe the problem was that `spatialimage` may not be available sometimes, but it could be just laziness.

Reply:
Yes, actually laziness:
```suggestion
        self.mapped.header.set_xyzt_units(spatialimage.header.get_xyzt_units())
```